charts/linkerd2/templates/namespace.yaml

```diff
@@ -13,5 +13,6 @@ metadata:
     {{.ProxyInjectAnnotation}}: {{.ProxyInjectDisabled}}
   labels:
     {{.LinkerdNamespaceLabel}}: "true"
```

Comment:
Can we remove this label?

Reply:
Sorry for the back-and-forth. Can we add this label back to this `namespace.yaml` file? I got carried away because deleting code excites me. `{{.LinkerdNamespaceLabel}}` is essentially `linkerd.io/is-control-plane`, and as mentioned earlier, it's still being used by the integration test suite to do post-test clean up.

Reply:
No worries, adding back now :)

Reply:
Opppss... there are some unit test failures. Can you run `go test ./cli/... --update` to update the test fixtures? :pray:

Reply:
Tests are :heavy_check_mark: again! Thanks!